frontend/src/metabase/components/ExplicitSize.tsx

```diff
@@ -77,11 +77,16 @@ function ExplicitSize<T extends BaseInnerProps>({
       }
 
       _getElement() {
-        let element = ReactDOM.findDOMNode(this);
-        if (selector && element instanceof Element) {
-          element = element.querySelector(selector) || element;
+        try {
+          let element = ReactDOM.findDOMNode(this);
+          if (selector && element instanceof Element) {
+            element = element.querySelector(selector) || element;
+          }
+          return element instanceof Element ? element : null;
+        } catch (e) {
+          console.error(e);
```

Comment:
Do we have a practice on how we handle console logs? Do we have a logger where the logs goes to? I think this might show the error in the SDK user's console, which might not be what they expected.

Reply:
Not right now. I think we could stick with this solution, and in the future we could add a debug flag to the SDK config that enables our console.error messages

Reply:
Sounds good! I was thinking of an environment check too, i.e. only show `console.error` when we are doing development builds.

Reply:
We could even do something referenced here for our embedding webpack config:
https://github.com/webpack-contrib/terser-webpack-plugin/issues/57